sphinx/source/docs/user_guide/charts.rst

```diff
@@ -559,22 +607,52 @@ Interface outputs
 -----------------
 
 As with the low and middle level ``Bokeh`` plotting APIs, in ``bokeh.charts``,
-we also support the chart output to a file::
+we also support the chart output to::
+
+* a file.
 
-    hist = Histogram(distributions, bins=50, filename="my_plot")
+    hist = Histogram(distributions, bins=50, filename="hist.html")
+    hist.show()
+
+    # or use
+    from bokeh.plotting import output_file, show
+    output_file('hist.html')
+    show(hist)
 
-* ``filename``, string type, the name of your chart.
 
-to the ``bokeh-server``::
+* to ``bokeh-server``::
 
     hist = Histogram(distributions, bins=50, server=True)
+    hist.show()
+
+    # or use
+    from bokeh.plotting import output_server, show
+    output_server('hist')
+    show(hist)
 
-* ``server``, string type, the name of your chart in the server.
 
-and to the IPython notebook::
+
+* to IPython notebook::
 
     hist = Histogram(distributions, bins=50, notebook=True)
+    hist.show()
 
-* ``notebook``, bool type, if you want to output (or not) to the notebook.
+    # or use
+    from bokeh.plotting import output_notebook, show
+    output_notebook()
+    show(hist)
 
 .. note:: You can output to any or all of these 3 possibilities because, right now, they are not mutually exclusive.
+
+
+
+.. _charts_builders:
+
+Chart Builders
+--------------
+
+Since 0.8 release `Charts` creation is streamlined by specific
+objects called Builders. Builders are convenience classes that create
+all computation, validation and low-level geometries needed to render a High Level
+Chart. This provides clear pattern to easily extend the `Charts` interface
+with new charts. For more info about this refer to the
```

Comment:
grrr... yes. Ideally I wanted to write a "how to" guide to build new charts.. but won't make in time for the release. Probably best is to refer to the references for now.